Resources/assets/js/adapter/fancytree.js

```diff
@@ -87,11 +87,7 @@ export class FancytreeAdapter {
             };
 
             for (let actionName in actions) {
-                if (!actions.has(actionName)) {
```

Comment:
I think this is from a leftover of using `actions.hasOwnProperty(actionName)` when I wrote this code in ES5. It makes sense in ES5, doesn't make much sense in ES6 indeed.